tests/Oryx.BuildImage.Tests/VersionInformationTest.cs

```diff
@@ -29,7 +29,7 @@ public VersionInformationTest(ITestOutputHelper output)
             _dockerCli = new DockerCli();
         }
 
-        [Fact, Trait("category", "latest")]
+        [SkippableFact, Trait("category", "latest")]
```

Comment:
Will `SkippableFact` go skipping these test if they failed, any reason we add it and should we revert it?

Reply:
@qianz2  No, it will still fail if it exits with anything other than a skip exception. All this does is allow us to run it locally (where the test returns a Skip exception) and the test will mark it as skipped instead of failed when it receives the skip exception